src/services/notifications/index.ts

```diff
@@ -520,3 +541,68 @@ export const NotificationsService = (logger: Logger): INotificationsService => {
     }),
   }
 }
+
+export const getPushNotificationContent = ({
+  type,
+  amountBaseCurrency,
+  baseCurrency,
+  amountDisplayCurrency,
+  displayCurrency,
+  userLanguage,
+}: {
+  type: NotificationType | "balance"
+  amountBaseCurrency: Satoshis | UsdCents
```

Comment:
We've been trying to shift to using `PaymentAmount<WalletCurrency>` type in new code in place of `Satoshis`/`UsdCents`. This method may be a good candidate for going with the new convention. Single `paymentAmount` property would replace both `amountBaseCurrency` and `baseCurrency` args.

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/1338/commits/487f3ddca353e8af8335487f89cd39fa69d183cd also `DisplayPaymentAmount` was added